typescript-timers/interval.ts

```diff
@@ -0,0 +1,13 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+if (!$countdownDisplay) throw new Error('$countdownDisplay query failed');
+
+let count = 4;
+
+const intervalId = setInterval(() => {
+  count--;
+  $countdownDisplay.textContent = `${count}`;
```

Comment:
Do we always want this to happen? By doing this, it would show 0 for a slight second, and then change to the text content.